### DISCUSSION TITLE
[Unstable] Transitions between levels break the game in campaign

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14592] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When transitioning from a level to another you'll get stuck on loading screen. You can switch characters during said screen and it will repeat the error. You can type `mainmenu` in console to go back to the menu, but otherwise stuck

https://github.com/user-attachments/assets/45e97ebb-4ab2-4004-8c3e-714341c4719a

### Reproduction steps

Start campaign
Leave the location

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/619c67a48ceac86d0be120480b2e49dacc6e4e28 (commit in our private repo), coming up in the next unstable build!

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
